lib/classifier-reborn/backends/bayes_redis_backend.rb

```diff
@@ -1,4 +1,8 @@
-require 'redis'
+begin
+  require 'redis'
+rescue LoadError
+  puts 'The redis gem is required to use the redis backend.'
```

Comment:
With more complex systems, I would recommend indicating where this comes from. This message should contain `classifier-reborn` somewhere in there.

Additionally, the user experience of wanting redis and not fulfilling the requirements should be considered. I think a hard `raise` would be the most direct way to ensure if the user wants to use redis, they get redis.